changelog/v1.18.19/dfp_circuit_breaker.yaml

```
@@ -0,0 +1,7 @@
+changelog:
+  - type: FIX
+    issueLink: https://github.com/solo-io/solo-projects/issues/8094
+    resolvesIssue: false
+    description: >-
+      For clusters generated for the dynamic forward proxy, the circuit breakers are now generated from the dfp config, if provided,
+      otherwise from the settings. If the circuitBreakers are not provided in either location, nenvoy will use the default circuit breakers.
```

```suggestion
      otherwise from the settings. If the circuitBreakers are not provided in either location, envoy will use the default circuit breakers.
```
Mayb also mention the default value?

---

Added a link to the envoy docs with the defaults